pkg/manifests/manifests.go

```diff
@@ -219,6 +220,10 @@ var (
 	PrometheusTrustedCABundlePath = PrometheusTrustedCABundleDir + "ca-bundle.crt"
 )
 
+var (
+	IBMCloudPlatformType configv1.PlatformType = "IBMCloud"
```

Comment:
Don't redefine constant as vars, that's horribly dangerous.

Reply:
Thx! fixed